schemas/common/principalprofile.example.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "xdm:provider": {
```

Comment:
I saw the example of https://github.com/adobe/xdm/blob/master/schemas/common/principal.example.1.json to arrive at this. I think xdm:provider should be just a URL but I am not sure if I can decide such a value on my own.